ppb/systems/renderer.py

```diff
@@ -234,9 +246,10 @@ def compute_rectangles(self, texture, game_object, camera):
 
         return src_rect, dest_rect, ctypes.c_double(-game_object.rotation)
 
-    def target_resolution(self, width, height, game_unit_size):
+    @staticmethod
+    def target_resolution(width, height, game_object_size, pixel_ratio):
         values = [width, height]
         short_side_index = width > height
-        target = self.pixel_ratio * game_unit_size
+        target = pixel_ratio * game_object_size
```

Comment:
Not actually a fan of this rename. The old name was more correct and we're not moving away from game units as a concept.

Reply:
Oh, yeah, sorry, I inlined this because pixel_ratio isn't directly a renderer concern anymore, then realized we had a test around this. Can revert.

Reply:
Yes please.